### PR TITLE
new bunny

### PR DESCRIPTION
**Pull request by TaylorHartsfield**
Added Buns Bunner to the zoo.

### CODE REVIEW

File: people2.js [hunk 1]
```diff
@@ -6,5 +6,6 @@ const PEOPLE = [
   { animal: 'lion', firstName: 'Pash', favoriteFood: 'burger', show: 'Bojack Horseman'},
   { animal: 'parrot', firstName: 'Peri', favoriteFood: 'couscous', show: 'The Walking Dead'}, 
   { animal: 'wolf', firstName: 'Wolfie', middleName: 'Leopold', favoriteFood: 'french fries', show: 'The Good Place'},
-  { animal: 'deer', firstName: 'Lily', middleName: 'Jane', show: 'Amazing Race'}
+  { animal: 'deer', firstName: 'Lily', middleName: 'Jane', show: 'Amazing Race'},
```
Comment (TaylorHartsfield) [inline people2.js:9]: @dargaCode Is this the correct information?

Comment (dargaCode) [reply people2.js:9]: @TaylorHartsfield looks great!

File: people2.js [hunk 2]
```diff
@@ -6,5 +6,6 @@ const PEOPLE = [
...
+  { animal: 'bunny', firstName: 'Buns', middleName: 'Bunner'}
```
Comment (dargaCode) [inline people2.js:10]: Missing favorite tv show

File: people2.js [hunk 3]
```diff
@@ -7,5 +7,5 @@ const PEOPLE = [
   { animal: 'parrot', firstName: 'Peri', favoriteFood: 'couscous', show: 'The Walking Dead'}, 
   { animal: 'wolf', firstName: 'Wolfie', middleName: 'Leopold', favoriteFood: 'french fries', show: 'The Good Place'},
   { animal: 'deer', firstName: 'Lily', middleName: 'Jane', show: 'Amazing Race'},
-  { animal: 'bunny', firstName: 'Buns', middleName: 'Bunner'}
+  { animal: 'bunny', firstName: 'Buns', middleName: 'Bunner', show: 'Survivor'}
```
Comment (TaylorHartsfield) [inline people2.js:10]: added bunners show, @dargaCode

**Comment by dargaCode** [comment]
LGTM!

**Review by dargaCode — APPROVED** [review]
Looks good to me!

**Pull request merged**